develop/tutorials/articles/14-themes-and-layout-templates/03-portlets-and-themes/01-theming-portlets.markdown

```diff
@@ -1,17 +1,26 @@
 # Theming Portlets [](id=theming-portlets)
 
-@product@ themes can provide additional styles to a portlet. You can change the
-markup for the portlet containers by modifying the `portlet.ftl` file.
+Themes can provide additional styles for your apps. You can change the markup 
+for portlet containers by modifying the theme's `portlet.ftl` file.
 
 This tutorial demonstrates how to style portlets with your themes.
 
 ## Portlet FTL [](id=portlet-ftl)
-<!--It doesn't seem to me that this section
-is giving instructions on how to do anything. I'm told that descriptive texts
-of this sort belong in /reference, not /tutorials-NR-->
-Here is a quick look at the default 
+
+Although you can individually style a portlet via the theme's CSS or the
+portlet's *Look and Feel Configuration* menu,<!--Can we link to docs on these
```

Comment:
The Look and Feel configuration menu will be in the User Guide docs once they're published. I can link to those. To style a portlet's CSS, the user would have to overwrite the default styles provided by the classes defined in the portlet.ftl file. We don't have any docs on how to do that. That would be up to the dev to figure out the specific styling. I don't think we can do more than saying what we do: 

> To provide a custom style for all portlets, use the CSS classes in this file for the various container elements...

Reply:
Ok. Thanks.